include/follow.php

```diff
@@ -82,6 +82,11 @@ function new_contact($uid,$url,$interactive = false) {
 		return $result;
 	}
 
+	if (blocked_url($url)) {
+		$result['message'] = t('Blocked domain');
```

Comment:
Users can act. They can talk to their admins about the reason why an entire domain was blocked. Maybe even agree with the reasoning and move forward to another domain when they disagree with the reasoning of the admin in that topic.